content/utilities/map-export/plugin.js

```diff
@@ -110,7 +112,40 @@ class Plugin {
       this.status.style.color = 'red';
     }
   }
-
+  
+  onDownload = async () => {
+    let chunks = ui.getExploredChunks();
+    let chunksAsArray = Array.from(chunks);
+    if (this.beginCoords && this.endCoords) {
+      let begin = {
+        x: Math.min(this.beginCoords.x, this.endCoords.x),
+        y: Math.max(this.beginCoords.y, this.endCoords.y),
+      };
+      let end = {
+        x: Math.max(this.beginCoords.x, this.endCoords.x),
+        y: Math.min(this.beginCoords.y, this.endCoords.y),
+      };
+      chunksAsArray = chunksAsArray.filter((chunk) => {
+        return this.intersectsXY(chunk, begin, end);
+      });
+    }
+    try {
+      let map = JSON.stringify(chunksAsArray);
+      var blob = new Blob([map], { type: 'text/plain' }),
+          anchor = document.createElement('a');
+      anchor.download = "hello.txt";
```

Comment:
This would be something like `map.json`
```suggestion
      anchor.download = "map.json";
```